### PR TITLE
Update app.js

### PR DESCRIPTION
**Pull request by dikshant19**

### CODE REVIEW

File: sample-js-app/app.js [hunk 1]
```diff
@@ -134,6 +134,10 @@ app.post("/delete", function(req, res){
     }
 });
 
+app.get("/testAPI", function(req, res){
```
Comment (prk54) [inline sample-js-app/app.js:137]: Consider adding a comment to explain the purpose of this route.

Comment (dikshant19) [reply sample-js-app/app.js:137]: what?

File: sample-js-app/app.js [hunk 2]
```diff
@@ -133,6 +133,10 @@ app.post("/delete", function(req, res){
       });
```
Comment (prk54) [inline sample-js-app/app.js:133]: Consider removing the extra space at the end of the line.

File: sample-js-app/app.js [hunk 3]
```diff
@@ -133,6 +133,10 @@ app.post("/delete", function(req, res){
...
     }
 });
+// API call
```
Comment (prk54) [inline sample-js-app/app.js:136]: Add a comment to describe the purpose of this API call.

File: sample-js-app/app.js [hunk 4]
```diff
@@ -134,6 +134,27 @@ app.post("/delete", function(req, res){
     }
 });
 
+
```
Comment (prk54) [inline sample-js-app/app.js:137]: The parameter name in the route handler function is 'testList', but it is being accessed as 'customListName'. Please ensure consistent naming.

File: sample-js-app/app.js [hunk 5]
```diff
@@ -134,6 +134,27 @@ app.post("/delete", function(req, res){
...
+app.get("/:testList", (req, res) => {
+  let customListName = _.capitalize(req.params.customListName);
```
Comment (prk54) [inline sample-js-app/app.js:139]: The variable 'customListName' is being capitalized using the '_.capitalize' function. However, this function is not defined in the code. Please make sure to import the necessary library or define the function before using it.

File: sample-js-app/app.js [hunk 6]
```diff
@@ -134,6 +134,27 @@ app.post("/delete", function(req, res){
...
+  List.findOne({name: customListName}, function(err, foundList){
+      if (!foundList) {
+        //Create a new list
```
Comment (prk54) [inline sample-js-app/app.js:142]: The comment 'Create a new list' is unnecessary and does not provide any additional information. Please remove the comment.

File: sample-js-app/app.js [hunk 7]
```diff
@@ -134,6 +134,27 @@ app.post("/delete", function(req, res){
...
+        let list = new List({
+          name: customListName,
+          items: defaultItems
+        });
+
+        list.save();
```
Comment (prk54) [inline sample-js-app/app.js:148]: The 'list.save()' method is being called without any error handling. It is recommended to handle errors when saving data to the database. Please consider adding error handling for this operation.

**Pull request closed without merging**